### DISCUSSION TITLE
How can I set the color for a status separator?

### DISCUSSION THREAD

**Discussion by mrt181** [catppuccin/tmux#397] [Help] [answered]
I need to color the middle separator because it defaults to the wrong color (black).

![image](https://github.com/user-attachments/assets/bec3304c-26cd-48a3-978d-6d482670b9f3)

![image](https://github.com/user-attachments/assets/6a791842-edad-4cff-9c38-e1760e68016d)

How can I do this?

**Comment by kjnsn** [comment]
The separator is green. Set the background appropriately using an embedded style 

See the tmux man page for details

**Comment by mrt181** [comment]
Why is it green?
It appears very dark.
You can see the difference when I remove the middle separator
![image](https://github.com/user-attachments/assets/3e417801-74d6-44f8-bce3-89a67c08e633)

**Reply by kjnsn** [reply]
Ah sorry, it's black because the current foreground is crust

**Comment by kjnsn** [comment, marked as answer]
So it's not defaulting to the wrong colour, everything is working exactly as intended

**Comment by mrt181** [comment]
as a reference for my future self, this is how I solved this:

```
set -g @catppuccin_status_middle_separator "#[fg=#{@thm_surface_0}]"
```

![image](https://github.com/user-attachments/assets/10e00fe0-440b-410f-9410-a2e1738e9e14)
